libs/wingc/src/lsp/rename_visitor.rs

```diff
@@ -168,6 +207,40 @@ impl<'a> Visit<'a> for RenameVisitor<'a> {
 				symbol: var_name.clone(),
 				references: vec![],
 			}),
+			//TODO: to be handled in a following PR, renaming interface fields is not supported yet
+			// StmtKind::Interface(c) => {
+			// for field in &c.methods {
+			// 	self.linked_symbols.push(LinkedSymbol {
+			// 		symbol: field.0.clone(),
+			// 		references: vec![],
+			// 	})
+			// }
+			// }
+			StmtKind::Struct(s) => {
+				for field in &s.fields {
+					self.linked_symbols.push(LinkedSymbol {
+						symbol: field.name.clone(),
+						references: vec![],
+					})
+				}
+			}
+			StmtKind::Class(c) => {
+				for (m, ..) in &c.methods {
+					if m.name == "new" {
```

Comment:
The constants `CLASS_INIT_NAME` and `CLASS_INFLIGHT_INIT_NAME` can be used here to check for both types of constructors